WordPress/gutenberg
# Migrate list view E2E tests to Playwright

## What?
Migrate List View E2E tests to Playwright.

## Why?
- Done as a part of the Playwright migration project: https://github.com/WordPress/gutenberg/issues/38851 (priority list),
- Should eventually resolve all [related flaky reports](https://github.com/WordPress/gutenberg/issues?q=is%3Aopen+is%3Aissue+label%3A%22%5BType%5D+Flaky+Test%22+%22specs%2Feditor%2Fvarious%2Flist-view.test.js%22).

## How?
- Used accessible selectors where possible,
- Ensured the UI has caught up when navigating with keyboard,
- Reworded some test titles for consistency,
- Added some comments for clarity.

## Testing Instructions

The output of the following should be all ✅: 

```
npm run test:e2e:playwright -- list-view.spec.js
```
